packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -89,11 +89,8 @@ function buildObjEntityFromProperty(
   entry: ApiEntryPoint,
   p: ApiPropertySignature
 ) {
-  const typeExcerpt = p.propertyTypeExcerpt;
-
-  const type = typeExcerpt.text;
-  const typeName = typeExcerpt.spannedTokens[0].text;
-  const apiInterface = findApi(entry, type) as ApiInterface;
+  const typeName = p.propertyTypeExcerpt.spannedTokens[0].text;
```

Comment:
I hit the same bug as you @btaillon. I was searching for the interface using the full type `typeExcerpt.text` which included the `[]` in the case of Interface[] types. With this change, we now search for the interface name, without the square brackets.